postgre-client/src/temp_index_client.rs

```diff
@@ -245,22 +265,42 @@ impl AssetIndexStorage for TempClient {
             assets_table: format!("{}assets_v3", TEMP_INDEXING_TABLE_PREFIX),
             creators_table: format!("{}asset_creators_v3", TEMP_INDEXING_TABLE_PREFIX),
             authorities_table: format!("{}assets_authorities", TEMP_INDEXING_TABLE_PREFIX),
-            fungible_tokens_table: format!("{}fungible_tokens", TEMP_INDEXING_TABLE_PREFIX),
         };
         self.pg_client
-            .upsert_batched(&mut transaction, table_names, updated_components)
+            .upsert_batched_non_fungible(&mut transaction, table_names, updated_components)
             .await?;
         self.pg_client.commit_transaction(transaction).await
     }
 
-    async fn update_last_synced_key(&self, last_key: &[u8]) -> Result<(), IndexDbError> {
+    async fn update_fungible_asset_indexes_batch(
+        &self,
+        fungible_asset_indexes: &[FungibleAssetIndex],
+    ) -> Result<(), IndexDbError> {
+        let mut c = self.pooled_connection.lock().await;
+        let mut transaction = c.begin().await?;
+
+        self.pg_client
+            .upsert_batched_fungible(
```

Comment:
This method will try inserting in the non-temp table. As the table name is hardcoded in `upsert_batched_fungible`
As an alternative - we could deprecate the temp_index_client alltogether as we're not using it and it didn't prove to be more performant compared to a regular sync